app/models/accounting/transaction.rb

```diff
@@ -208,6 +207,14 @@ def net_debit_for_account(account_id)
     end
   end
 
+  def disbursement_fields
+    if loan_transaction_type_value == "disbursement"
+      errors.add(:disbursement_type) unless disbursement_type.present?
+      errors.add(:qb_vendor_id) unless qb_vendor_id.present?
```

Comment:
Not sure what is going on here as there is no error message. Is this meant to be a presence validation, like 'can't be blank'? If so, can we just list those at the top and use `if` to apply them conditionally?